docker/Dockerfile.pytiramisu

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2020 R. Tohid (@rtohid)
```

Comment:
Instead of adding a new dockerfile, can we merge the capabilities of this one with the existing one? Thanks.